lib/ui/views/profile/edit_profile_view.dart

```diff
@@ -40,22 +55,36 @@ class _EditProfileViewState extends State<EditProfileView> {
       initialValue: _name,
       validator: (value) => value.isEmpty ? "Name can't be empty" : null,
       onSaved: (value) => _name = value.trim(),
+      onFieldSubmitted: (_) =>
```

Comment:
The named parameter 'onFieldSubmitted' isn't defined.
Try correcting the name to an existing named parameter's name, or defining a named parameter with the name 'onFieldSubmitted'.